pkg/statser/statser_internal.go

```diff
@@ -8,12 +8,18 @@ import (
 	"github.com/atlassian/gostatsd"
 )
 
-// InternalHandler is an interface to dispatch metrics to.  Exists
+// InternalMetricHandler is an interface to dispatch metrics to.  Exists
 // to break circular dependencies.
-type InternalHandler interface {
+type InternalMetricHandler interface {
 	DispatchMetric(ctx context.Context, m *gostatsd.Metric) error
```

Comment:
May remove variable names - types are descriptive enough.

Reply:
I'm a big fan of variable names - it makes it easier to copy and paste the interface definition.  Plus general verbosity is rarely bad.